skglm/penalties/separable.py

```diff
@@ -273,6 +283,89 @@ def alpha_max(self, gradient0):
         return np.max(np.abs(gradient0))
 
 
+class WeightedMCPenalty(BasePenalty):
+    """Minimax Concave Penalty (MCP), a non-convex sparse penalty.
+
+    Notes
+    -----
+    With :math:`x >= 0`:
+
+    .. math::
+        "pen"(x) = {(alpha x - x^2 / (2 gamma), if x =< alpha gamma),
+                    (gamma alpha^2 / 2        , if x > alpha gamma):}
+    .. math::
+        "value" = sum_(j=1)^(n_"features") "pen"(abs(w_j))
```

Comment:
Weights are missing in the formula, right?
```suggestion
        "value" = sum_(j=1)^(n_"features") "weights"_j "pen"(abs(w_j))
```